src/mixins/filing-template-mixin.ts

```diff
@@ -19,8 +21,8 @@ export default class FilingTemplateMixin extends Vue {
   // Global Getters
   @Getter isTypeBcomp!: GetterIF
   @Getter getApprovedName!: string
-
-  // Global Actions
+  @Getter getBusinessIdentifier!: string
```

Comment:
Pls restore blank line for nice code readability :)

Reply:
okie dokie